transport.go

```diff
@@ -45,7 +48,6 @@ func (c *GRPCClient) SendToIngest(ctx context.Context, req *ingest.IngestRequest
 		log.From(ctx).Error("speakeasy-sdk: failed to create grpc connection", zap.Error(err))
 		return
 	}
-	defer conn.Close()
```

Comment:
we will need to expose a Close() method (and the appropriate documentation) on the SDK to allow the connection to be closed for clean shutdowns of integrators servers